proposals/simd/SIMD.md

```diff
@@ -820,3 +842,21 @@ are interpreted as signed integers.
 
 Converts low or high half of the smaller lane vector to a larger lane vector,
 sign extended or zero (unsigned) extended.
+
+```python
+def S.widen_low_T(ext, a):
+    result = S.New()
+    for i in range(S.Lanes):
+        result[i] = ext(a[S.Lanes + i])
+
+def S.widen_high_T(ext, a):
+    result = S.New()
+    for i in range(S.Lanes):
+        result[i] = ext(a[i])
+
+def S.widen_low_T_s(a):
+    return S.widen_low_T(Sext, a)
+
```

Comment:
Are we missing S.widen_low_T_u and S.widen_high_T_s?

Reply:
Oh yeah good point. Will update once we resolve the high/low terminology above.